items.go

```diff
@@ -5,20 +5,29 @@ import (
 	"net/http"
 )
 
+// AddItem adds new item of the given itemId to the items catalog.
+//
+// All the item properties for the newly created items are set to null.
 func AddItem(itemId string) Request {
 	return Request{
 		Path:   fmt.Sprintf("/items/%s", itemId),
 		Method: http.MethodPut,
 	}
 }
 
+// DeleteItem deletes an item of the given itemId from the catalog.
+//
+// If there are any purchases, ratings, bookmarks, cart additions, or detail views of the item present in the database,
+// they will be deleted in cascade as well. Also, if the item is present in some series, it will be removed from all the
```

Comment:
?? How it is possible that they are deted in cascade? I think it is only true, when `WithCascade` request option is given. I would not believe that it works as it is written here

Reply:
Copied verbatim from the official documentation: https://docs.recombee.com/api.html#delete-item

Reply:
Why do we have copied docs?

I do not really understand the reason behind it. Rather we can have no docs and just push reference to this docs.